net.c

```diff
@@ -63,7 +63,15 @@ int redisNetRead(redisContext *c, char *buf, size_t bufcap) {
         if ((errno == EWOULDBLOCK && !(c->flags & REDIS_BLOCK)) || (errno == EINTR)) {
             /* Try again later */
             return 0;
-        } else {
+        }
+        #ifdef _WIN32
```

Comment:
This doesn't need an `#ifdef`

Reply:
also, fix brace on same line

Reply:
> also, fix brace on same line

i fixed it.

Reply:
> This doesn't need an `#ifdef`

i removed #ifdef ~ #endif

Reply:
You're still using 8 space indentation.. are you using a proper editor? :)

Reply:
My default indent setting(VS2017) is a tab. :]
I (really) have fixed all formatting mistakes.